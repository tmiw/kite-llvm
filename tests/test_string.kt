"A"|asc|print;
"G"|bool|print; 
""|bool|print; 
"5.67"|float|print; 
"5"|int|print; 
"0123456789"|length|print; 
"HELLO"|lower|print; 
"   HELLO"|ltrim|print; 
"Hello"|print; 
"HELLO   "|rtrim|print; 
"   HELLO   "|str|print; 
"   HELLO   "|trim|print; 
"hello"|upper|print; 
