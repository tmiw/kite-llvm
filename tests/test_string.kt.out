65
true
false
5.67
5
10
hello
HELLO
Hello
HELLO
   HELLO   
HELLO
HELLO
